rbs/prism/CommentsAssociatorPrism.cc

```diff
@@ -441,6 +441,15 @@ template <typename PrismNode> void CommentsAssociatorPrism::walkAssignmentNode(p
     consumeCommentsInsideNode(untypedNode, "assign");
 }
 
+template <typename PrismNode>
+void CommentsAssociatorPrism::walkReceiverWriteNode(pm_node_t *untypedNode, std::string_view label) {
```

Comment:
I think both of these walker functions can go to the anonymous namespace, in other code reviews they usually want to keep the public surface of the function as little as possible.

Reply:
Then you'd have to pass a `CommentsAssociatorPrism` as a param. Which is worse?